src/main/scala/treadle/executable/BlackBoxCycler.scala

```diff
@@ -5,6 +5,16 @@ package treadle.executable
 import firrtl.ir.Info
 import treadle.ScalaBlackBox
 
+/**
+  * Implements an assigner that can be scheduled to publish clock transitions
+  * to specific black box implementations
+  * @param symbol symbol name of instance
```

Comment:
The instance is a pointer to a scala object which is the implementation of the black box, i.e. the specific code that is called when a black box's inputs are changed or output are used. That black box has a symbol in order to reference to it, the  symbol is there because it is required for all Assigners largely for debugging purposes.